getting-started/README.md

```diff
@@ -1,50 +1,50 @@
       - [Initialize a meter and collect metrics](#initialize-a-meter-and-collect-metrics)
       - [Initialize and register a metrics exporter](#initialize-and-register-a-metrics-exporter)
 
-## Tracing Your Application with OpenTelemetry
+## Trace your application with OpenTelemetry
 
-([link to TypeScript version](ts-example/README.md#tracing-your-application-with-opentelemetry))
+([link to TypeScript version](ts-example/README.md#trace-your-application-with-opentelemetry))
 
-This guide assumes you are going to be using Zipkin as your tracing backend, but modifying it for Jaeger should be straightforward.
+This guide assumes you're using Zipkin as your tracing backend, but modifying it for Jaeger should be straightforward.
 
-An example application which can be used with this guide can be found in the [example directory](example). You can see what it looks like with tracing enabled in the [traced-example directory](traced-example).
+You can find an example application to use with this guide in the [example directory](example). See what it looks like with tracing enabled in the [traced-example directory](traced-example).
 
-### Setting up a Tracing Backend
+### Set up a tracing backend
 
-([link to TypeScript version](ts-example/README.md#setting-up-a-tracing-backend))
+([link to TypeScript version](ts-example/README.md#set-up-a-tracing-backend))
 
-The first thing we will need before we can start collecting traces is a tracing backend like Zipkin that we can export traces to. If you already have a supported tracing backend (Zipkin or Jaeger), you can skip this step. If not, you will need to run one.
+The first thing you need before you can start collecting traces is a tracing backend like Zipkin that you can export traces to. If you already have a supported tracing backend (Zipkin or Jaeger), you can skip this step. If not, you need to run one.
 
-In order to set up Zipkin as quickly as possible, run the latest [Docker Zipkin](https://github.com/openzipkin/docker-zipkin) container, exposing port `9411`. If you can’t run Docker containers, you will need to download and run Zipkin by following the Zipkin [quickstart guide](https://zipkin.io/pages/quickstart.html).
+To set up Zipkin as quickly as possible, run the latest [Docker Zipkin](https://github.com/openzipkin/docker-zipkin) container, exposing port `9411`. If you can’t run Docker containers, you need to download and run Zipkin by following the Zipkin [quickstart guide](https://zipkin.io/pages/quickstart.html).
 
 ```sh
 docker run --rm -d -p 9411:9411 --name zipkin openzipkin/zipkin
 ```
 
-Browse to <http://localhost:9411> to ensure that you can see the Zipkin UI.
+Browse to <http://localhost:9411> to make sure you can see the Zipkin UI.
 
 <p align="center"><img src="./images/zipkin.png?raw=true"/></p>
 
-### Trace Your NodeJS Application
+### Trace your NodeJS application
```

Comment:
I reckon we can use Title Case for headers

Reply:
nvm as per https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings pointed out by @snyder114